ci-operator/jobs/cri-o/cri-o/cri-o-cri-o-main-periodics.yaml

```diff
@@ -26,6 +26,17 @@ periodics:
     ci.openshift.io/generator: prowgen
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
   name: periodic-ci-cri-o-cri-o-main-periodics-setup-fedora-periodic
+  reporter_config:
+    slack:
+      channel: '#forum-node-jira'
+      job_states_to_report:
+      - success
+      - failure
+      - error
+      report_template: '{{if eq .Status.State "success"}} :rainbow: Job *{{.Spec.Job}}*
```

Comment:
it's possible success notifications may get noisy, but we can see how we feel after a bit